project/merge.go

```diff
@@ -50,8 +50,21 @@ func Merge(p *Project, bytes []byte) (map[string]*ServiceConfig, error) {
 		datas[name] = data
 	}
 
-	err = utils.Convert(datas, &configs)
-	return configs, err
+	if err := utils.Convert(datas, &configs); err != nil {
+		return nil, err
+	}
+
+	adjustValues(configs)
+	return configs, nil
+}
+
+func adjustValues(configs map[string]*ServiceConfig) {
+	// yaml parser turns "no" into "false" but that is not valid for a restart policy
+	for _, v := range configs {
+		if v.Restart == "false" {
+			v.Restart = "no"
+		}
+	}
```

Comment:
@dnephin I don't know how to proceed because a docker-compose.yml from docker compose will work in 1.5 and blow up in libcompose-cli.  It sounds to me like we need this change to stay compatible.  @joshwget is working on schema validation.  Maybe when that PR is done we can revisit this.